packages/codemods/src/transforms/emotion-to-compiled/emotion-to-compiled.tsx

```diff
@@ -42,11 +42,15 @@ const imports = {
 };
 
 const removeEmotionJSXPragma = (j: core.JSCodeshift, collection: Collection) => {
-  const commentCollection = collection.find(j.Comment);
+  // ast-types are incorrect and can't handle both line and block types
+  // Cast to the correct type so we get safety
+  const commentCollection: Collection<CommentLine | CommentBlock> = collection.find(
```

Comment:
Can you pass this to `find` instead? i.e. `comment.find<CommentLine | CommentBlock>`

Reply:
Unfortunately not, as there's no corresponding function type for j.Comment :/